pyblish/cli.py

```diff
@@ -355,7 +355,7 @@ def gui(ctx, package):
 
     with _cli_plugin(data=context.data) as plugin_path:
         environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
-            ctx.obj["plugin_paths"] + [plugin_path]
+            list(ctx.obj["plugin_paths"]) + [plugin_path]
```

Comment:
IMHO, maybe it's better to do this conversion at the beginning of the function, like
```python
def gui(ctx, package):

    environ = os.environ.copy()
    context = ctx.obj["context"]
    plugin_paths = list(ctx.obj["plugin_paths"])
    ...
```

Reply:
I think that's a good idea, but I think we can go even further, and convert those as soon as they come out of Click. Having a look now.